data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature

```diff
@@ -7067,8 +7067,8 @@ Feature: Data Import integration tests
     And headers headersUser
     When method GET
     Then status 200
-    And match response.entries[*].relatedHoldingsInfo[0].actionStatus == ["UPDATED","UPDATED","UPDATED","UPDATED","UPDATED","UPDATED"]
-    And match response.entries[*].relatedItemInfo[0].actionStatus == ["UPDATED","UPDATED","UPDATED","UPDATED","UPDATED","UPDATED"]
+    And match response.entries[*].relatedHoldingsInfo[0].actionStatus == ["DISCARDED","DISCARDED","DISCARDED","DISCARDED","DISCARDED","DISCARDED"]
+    And match response.entries[*].relatedItemInfo[0].actionStatus == ["DISCARDED","DISCARDED","DISCARDED","DISCARDED","DISCARDED","DISCARDED"]
```

Comment:
it looks like we must not change this verification because the test checks that holdings and item update. Also, during this run the test passed https://jenkins-aws.indexdata.com/job/folioRancher/job/folioQualityGates/job/folioCiQualityGates/1225/cucumber-html-reports/report-feature_222_1684211029.html